ngi_pipeline/engines/piper_ngi/launchers.py

```diff
@@ -432,16 +459,17 @@ def sbatch_piper_sample(command_line_list, workflow_name, project, sample,
     with open(sbatch_outfile, 'w') as f:
         f.write("\n".join(sbatch_text_list))
     LOG.info("Queueing sbatch file {} for job {}".format(sbatch_outfile, job_identifier))
+    slurm_job_id=7777777
```

Comment:
Yes, I'm still in the process of making sure that the piper engine was not impacted.